### PR TITLE
Lattice 910 data layer property summary

### PR DESCRIPTION
**Pull request by pckeyes**
Implementation of property summary data layer

### CODE REVIEW

**Comment by geekbeast** [comment]
Just use hyphen instead of underscores in branch names as convention.

File: src/main/java/com/openlattice/postgres/ResultSetAdapters.java [hunk 1]
```diff
@@ -833,4 +834,17 @@ public static Entity entity( ResultSet rs, Set<UUID> authorizedPropertyTypeIds )
...
     }
 
+    /*
+    make a method called propertySummary. This is where you will loop through the results of the ResultsSet
+    and turn it into a JSON. You will need to implement a class like Entity.java used in the above entity() method.
+    It will return a JSON like whatever Leland drew in your notebook.
+     */
+    public static PropertySummary propertySummary( ResultSet rs ) throws SQLException {
+        UUID id = id( rs );
+        UUID entityTypeID = (UUID) rs.getObject("entity_type_id");
```
Comment (geekbeast) [inline src/main/java/com/openlattice/postgres/ResultSetAdapters.java]: extract constants

File: src/main/java/com/openlattice/postgres/ResultSetAdapters.java [hunk 2]
```diff
@@ -833,4 +836,11 @@ public static Entity entity( ResultSet rs, Set<UUID> authorizedPropertyTypeIds )
         return new Entity( entityKeyId, data );
     }
 
+    public static PropertySummary propertySummary( ResultSet rs ) throws SQLException {
+        UUID entityTypeID = (UUID) rs.getObject(ENTITY_TYPE_ID_FIELD);
+        UUID entitySetId = (UUID) rs.getObject( ENTITY_SET_ID_FIELD );
+        int count = rs.getInt(COUNT);
```
Comment (geekbeast) [inline src/main/java/com/openlattice/postgres/ResultSetAdapters.java]: This should be a ```rs.getLong(...)```  see return type for ```count(*)``` here.

Comment (pckeyes) [reply src/main/java/com/openlattice/postgres/ResultSetAdapters.java]: Change made

**Pull request merged**